common/surface.c

```diff
@@ -891,7 +891,7 @@ l_surface_getRawPixel(lua_State *L)
 	int x = luaL_checkinteger(L, 2);
 	int y = luaL_checkinteger(L, 3);
 	int size = surf->format->BytesPerPixel;
-	Uint8 *ptr = (Uint8 *)surf->pixels + y * surf->pitch + x * size;
+	char *ptr = (char *)surf->pixels + y * surf->pitch + x * size;
 	lua_pushlstring(L, ptr, size);
```

Comment:
Alright; I'd prefer that expression. Can you rebase this branch to use that version, or else open a new pull request?